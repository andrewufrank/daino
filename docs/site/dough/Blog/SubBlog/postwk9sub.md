---
abstract: postwk9sub abstract A silly text not needing an abstract.
author: AUF
date: Jan. 4, 2019
keywords: test
title: postwk9sub title
version: publish
visibility: public
---
postwk9sub Some text without meaning;  postwk.



  An example post.