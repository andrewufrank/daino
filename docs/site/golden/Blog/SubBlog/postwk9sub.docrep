Docrep {meta1 = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/postwk9sub.md", dyLink = "Blog/SubBlog/postwk9sub.md", dyLang = DLenglish, dyTitle = "postwksub.md", dyAbstract = "A silly text not needing an abstract.", dyAuthor = "AUF", dyDate = Just "Jan. 4, 2019", dyKeywords = "test", dyBibliography = Nothing, dyStyle = Nothing, dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = False, dyIndexSort = Nothing, dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/postwk9sub.md", link = "Blog/SubBlog/postwk9sub.md", title = "postwksub.md", abstract = "A silly text not needing an abstract.", author = "AUF", date = "Jan. 4, 2019", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}}, pan1 = Pandoc (Meta {unMeta = fromList [("abstract",MetaInlines [Str "A",Space,Str "silly",Space,Str "text",Space,Str "not",Space,Str "needing",Space,Str "an",Space,Str "abstract."]),("author",MetaInlines [Str "AUF"]),("date",MetaInlines [Str "Jan.",Space,Str "4,",Space,Str "2019"]),("keywords",MetaInlines [Str "test"]),("pageTemplate",MetaInlines [Str "page3.yaml"]),("title",MetaInlines [Str "postwksub.md"])]}) [Para [Str "Some",Space,Str "text",Space,Str "without",Space,Str "meaning;",Space,Str "postwk."],Para [Str "An",Space,Str "example",Space,Str "post."]]}