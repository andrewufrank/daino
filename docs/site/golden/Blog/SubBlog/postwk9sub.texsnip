TexSnip {snipyam = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/postwk9sub.md", dyLink = "Blog/SubBlog/postwk9sub.md", dyLang = DLenglish, dyTitle = "postwksub.md", dyAbstract = "A silly text not needing an abstract.", dyAuthor = "AUF", dyDate = Just "Jan. 4, 2019", dyKeywords = "test", dyBibliography = Nothing, dyStyle = Nothing, dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = False, dyIndexSort = Nothing, dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/postwk9sub.md", link = "Blog/SubBlog/postwk9sub.md", title = "postwksub.md", abstract = "A silly text not needing an abstract.", author = "AUF", date = "Jan. 4, 2019", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}}, unTexSnip = "Some text without meaning; postwk.\n\nAn example post."}