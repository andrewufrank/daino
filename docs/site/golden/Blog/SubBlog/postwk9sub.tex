\documentclass[a4paper,10pt]{scrbook}
\usepackage{fontspec}
\usepackage[ngerman]{babel}
\usepackage{graphicx}
\usepackage{makeidx}
\usepackage{natbib}
\makeindex
\usepackage[colorlinks]{hyperref}
\providecommand{\tightlist}{%
\setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

\begin{document}

Some text without meaning; postwk.

An example post.


\printindex
\end{document}
